.env.example

```
@@ -15,12 +15,22 @@ AUTH_MIDDLEWARE_ENABLED=true
 NEXT_PUBLIC_AUTH_LEGACY_BRIDGE=true
 
 # AI assistant (Vercel AI SDK)
+# Preferred default_chat contract: switch model/provider by changing the
+# provider-prefixed gateway model id and redeploy.
+AI_DEFAULT_CHAT_MODEL=google/gemini-3.1-flash-lite-preview
+AI_GATEWAY_API_KEY=
+# Optional explicit pin. Resolver already defaults to gateway when a
+# provider-prefixed AI_DEFAULT_CHAT_MODEL is set and legacy Google envs are absent.
+AI_DEFAULT_CHAT_PROVIDER=gateway
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_DEFAULT_CHAT_PROVIDER key should go before the AI_GATEWAY_API_KEY key

(UnorderedKey)